app/views/support/responsible_bodies/show.html.erb

```diff
@@ -15,106 +15,141 @@
       <%= @responsible_body.name %>
     </h1>
 
-    <%- if @responsible_body.extra_mobile_data_requests.present? %>
-      <h2 class="govuk-heading-l govuk-!-margin-top-9">Mobile data requests</h2>
-      <%= render Support::ExtraMobileDataRequestsSummaryListComponent.new(requests: @responsible_body.extra_mobile_data_requests) %>
-    <%- end %>
-
-    <h2 class="govuk-heading-l govuk-!-margin-top-9">Users</h2>
-
-    <% if policy(User).new? %>
-      <%= govuk_button_link_to 'Invite a new user', new_support_responsible_body_user_path(@responsible_body) %>
-    <% end %>
-
-    <% if @users.present? %>
-      <% @users.each do |user| %>
-        <div class="user">
-          <h3 class="govuk-heading-m govuk-!-font-size-27 govuk-!-margin-bottom-2">
-            <%= govuk_link_to user.full_name, support_user_path(user) %>
-          </h3>
-          <% if policy(user).edit? %>
-            <p class="govuk-body">
-              <%= govuk_link_to "Edit user<span class=\"govuk-visually-hidden\"> #{user.full_name}</span>".html_safe, edit_support_user_path(user) %>
-            </p>
-          <% end %>
-          <%= render Support::UserPreviewSummaryListComponent.new(user: user) %>
-        </div>
-      <% end %>
-    <% else %>
-      <p class="govuk-body">None</p>
-    <% end %>
-
-    <% if @responsible_body.has_virtual_cap_feature_flags_and_centrally_managed_schools? %>
-      <div class="govuk-grid-row">
-          <div class="govuk-grid-column-two-thirds">
-            <h2 class="govuk-heading-l govuk-!-margin-top-9">Centrally managed schools and colleges</h2>
-            <p class="govuk-body"><%= @responsible_body.name %>:</p>
-            <ul id="responsible-body-centrally-managed-stats" class="govuk-list govuk-list--bullet">
-              <li>manages ordering for <%= centrally_managing_count_or_all_schools(responsible_body: @responsible_body) %> of its schools and colleges</li>
-              <li>has <%= what_to_order_allocation_list(allocations: @virtual_cap_pools) %> available to order right now</li>
-              <li>has ordered <%= what_to_order_state_list(allocations: @virtual_cap_pools) %> so far</li>
-            </ul>
+    <div class="govuk-tabs" data-module="govuk-tabs">
```

Comment:
Could we switch this out to use govuk_components? 
https://dfe-digital.github.io/govuk-components/#tabs

Reply:
Yes, but i think that's outside the scope of this ticket